src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1137,6 +1137,14 @@ class ApplicationManagerImpl
   protocol_handler::MajorProtocolVersion SupportedSDLVersion() const OVERRIDE;
 
  private:
+  /**
+   * @brief Stop services that failed to start
+   * @param app Application whose service should be stopped
+   * @param Service type that should be stopped
+   */
+  bool StopServicesForApp(
```

Comment:
@IGapchuk Can the name of this method be changed to something other than "Stop" since video and audio services are just being cleaned up. Maybe "HandleRejectedServiceStatus()"?

Reply:
@JackLivio  I agree with you, the `HandleRejectedServiceStatus` name looks better for the method.
Please, take a look at changes: 148c16bb64335b5be093c63d144ee5a48211d724

Reply:
@JackLivio  The comments have been processed. Please inform me when the review will be completed,`--fixup` commits should be squashed and fix branch should be rebased onto current develop. Thank you.